### ISSUE TITLE
Support for OpenAPI v3

### ISSUE THREAD

**Issue by fradav** [fsprojects/SwaggerProvider#189]
### Description

PetStore basic example fails to load v3 schema

### Repro steps

1. Create a `swagger.fsx` file

```fsharp
#r "nuget: SwaggerProvider, 1.0.0-beta5"
open SwaggerProvider

let [<Literal>] Schema = "https://petstore3.swagger.io/api/v3/openapi.json"

type PetStore = OpenApiClientProvider<Schema>
```

2. Run it with fsharp interactive 

```bash
dotnet fsi swagger.fsx
```

### Expected behavior

No error

### Actual behavior

```plain
/home/fradav/Dev/dotnet/fsharp/ldap/swagger.fsx(6,17): error FS3033: Le fournisseur de type 'SwaggerProvider.OpenApiClientTypeProvider' a signalé une erreur : Operation 'uploadFile' does not contain supported media types ["application/octet-stream"]
```

### Known workarounds

Stick with `v2` api.

### Related information 

* Linux
* SwaggerProvider version 1.0.0-beta5
* dotnet 5.0.203

**Comment by xperiandri** [comment]
I suppose it already supports it in `2.0.0-beta*`

**Comment by sergey-tihon** [comment]
no. it is a different issue

**Comment by sergey-tihon** [comment]
Fixed by #203
